test/Interop/SwiftToCxx/structs/zero-sized-struct-in-cxx.swift

```diff
@@ -1,11 +0,0 @@
-// RUN: %empty-directory(%t)
-// RUN: %target-swift-frontend %s -typecheck -module-name Structs -clang-header-expose-decls=all-public -emit-clang-header-path %t/structs.h
-// RUN: %FileCheck %s < %t/structs.h
-
-// CHECK: namespace Structs SWIFT_PRIVATE_ATTR SWIFT_SYMBOL_MODULE("Structs") {
-
-// CHECK-NOT: class SWIFT_SYMBOL({{.*}}) ZeroSizedStruct final {
-
-public struct ZeroSizedStruct {}
```

Comment:
Would it be reasonable to keep this test, but check that the struct is emitted with size=1 instead of not being emitted?

Reply:
Thank you for the quick review.
That's reasonable. I will work to add the test you described.